src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,83 @@
+import "./Profile.css";
+import SideBar from "../SideBar/SideBar";
+import { useState } from "react";
+
+function Profile() {
+  const [searchInput, setSearchInput] = useState("");
+  const [albums, setAlbums] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchInput(e.target.value);
+  };
+
+  const handleSearch = async () => {
+    try {
+      const accessToken = localStorage.getItem("spotify_access_token");
+
+      const response = await fetch(
+        `https://api.spotify.com/v1/search?type=album&q=${encodeURIComponent(
+          searchInput
+        )}`,
+        {
+          headers: {
+            Authorization: `Bearer ${accessToken}`,
+          },
+        }
+      );
+
+      if (response.ok) {
+        const data = await response.json();
+        setAlbums(data.albums.items);
+      } else {
+        console.error("Error fetching album data:", response.status);
+      }
+    } catch (error) {
+      console.error("Error searching albums:", error);
+    }
+  };
+  return (
+    <div className="profile">
+      <section className="profile__sidebar">
+        <SideBar />
+      </section>
+      <div className="profile__search">
+        <label className="profile__search_input" htmlFor="myInput">
+          Search :
+          <input
```

Comment:
If you have an `input` in the markup, then it should be inside a `form` tag, according to the semantics.